packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -134,4 +136,22 @@ export class ConfigUtil {
       : undefined;
     return username ? String(username) : undefined;
   }
+
+  public static async setDefaultUsernameOrAlias(usernameOrAlias: string): Promise<void> {
```

Comment:
I think configUtil.test.ts needs some tests to cover this new function.  Those tests should cover:

* That the cwd is switched to the project root if necessary (and switched back to the orig cwd after)
* That the core lib is being called correctly.  i.e., when called with <username>, Org.create is called
  with <username>, and then Config.set() is called with "<OrgConfigProperties.TARGET_ORG>, <username>", etc.